SQL/DDL.sql

```diff
@@ -1,67 +1,65 @@
-create table customer
-	(email		                varchar(30) not null, 
-	 password			        varchar(30) not null, 
-	 name		                varchar(30) not null,
-	 phone		                numeric(10,0),
-     address_street_num		    varchar(30), 
-     address_street_name		varchar(30), 
-     address_street_postal		varchar(30), 
+CREATE TABLE customer
+  (
+     email                 varchar(30) not null,
+     password              varchar(30) not null,
+     name                  varchar(30) not null,
+     phone                 numeric(10, 0),
+     address_street_num    varchar(30),
+     address_street_name   varchar(30),
+     address_street_postal varchar(30),
+     primary key (email)
+  );
 
-	 primary key (email)
-	);
+CREATE TABLE bank_account
+  (
+     bank_number numeric(10, 0) not null,
+     amount      int not null,
+     primary key (bank_number)
+  );
 
-create table bank_account
-	(bank_number                numeric(10,0) not null, 
-	 amount			            int not null,
-
-	 primary key (bank_number)
-	);
-
-create table book_order
-	(order_id		            numeric(10,0) not null, 
-	 email		                varchar(30) not null, 
-	 shipping_id                numeric(10,0) not null,
-
-	 primary key (order_id),
+CREATE TABLE book_order
+  (
+     order_id    numeric(10, 0) not null,
```

Comment:
Should probably be int and be an auto generating id. Could add serial tag for that